cli/src/commands/subgraph/commands/check.ts

```diff
@@ -41,6 +41,19 @@ export default (opts: BaseCommandOptions) => {
       };
     }
 
+    let ignoreErrors = false;
+    if (gitInfo) {
+      const integrationCheckResponse = await opts.client.platform.isGitHubAppInstalled(
+        {
+          gitInfo,
+        },
+        {
+          headers: baseHeaders,
+        },
+      );
+      ignoreErrors = integrationCheckResponse.isInstalled;
```

Comment:
Could we also log a message that the result depends on the PR check? This helps in debugging scenarios.

Reply:
Done